src/Core/Hook/Provider/IdentifiableObjectHookByFormTypeProvider.php

```diff
@@ -41,7 +41,9 @@ final class IdentifiableObjectHookByFormTypeProvider implements HookByFormTypePr
     public const FORM_TYPE_POSITION_IN_CONSTRUCTOR_OF_FORM_BUILDER = 0;
 
     public const FORM_BUILDER_HOOK_PREFIX = 'action';
-    public const FORM_BUILDER_HOOK_SUFFIX = 'FormBuilderModifier';
+    public const FORM_BUILDER_HOOK_SUFFIX_MODIFIER = 'FormBuilderModifier';
```

Comment:
All these const could be private no? In both cases this is a BC break that needs to be documented :trollface:

Reply:
Fixed